Adds TuxCare and OpenLogic blurbs

mattlasheboro opened the pull request:
Based on issue:

https://github.com/AlmaLinux/almalinux.org/issues/105

mattlasheboro commented:
**How it appears on small screens:**
![commercial-support-small-displays](https://github.com/AlmaLinux/almalinux.org/assets/10442336/b1b575e8-c601-4081-987b-f37c9d25c0ec)

**How it appears on larger screens:**
![commercial-support-large-displays](https://github.com/AlmaLinux/almalinux.org/assets/10442336/1315dd9e-eca4-40c4-9fc4-2241efc93f65)

bennyvasquez commented on layouts/index.html:
```diff
@@ -494,29 +494,32 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
+						</a>
+                     </div>
+					 <div class="col-12 col-md-8">
+						<p class="">AlmaCare is an enterprise-grade AlmaLinux support service delivered by one of the founding sponsors of the AlmaLinux Project, TuxCare, a division of CloudLinux Inc.</p>
```
```suggestion
						<p class="">Confidently deploy AlmaLinux and all your open source packages with enterprise-grade, 24/7/365 support backed by guaranteed SLAs. Keep your mission-critical apps running smoothly with unlimited tickets and direct access to experienced enterprise architects who put your business first. [Get Support](https://www.openlogic.com/solutions/enterprise-linux-support/almalinux)</p>
```

bennyvasquez commented:
@mattlasheboro this'll be perfect! But in the code can you add <!-- Cybertrust Japan --> sort of delineators between the sections? That'll help future editing, and making sure that all of the sections have the right content.

mattlasheboro commented:
@bennyvasquez Good idea, I added some spacing and a comment to try and increase the readability a bit.

The indentation is a bit wonky overall, some places use tabs and some use spaces but I believe I got everything lined up correctly.

I also removed the empty class attributes that I inadvertently left on the P tags.

codyro commented:
> The indentation is a bit wonky overall, some places use tabs and some use spaces but I believe I got everything lined up correctly.

The whole code base through a linter and then add it to our CI workflow for PR's. It's on the to do list :)

bennyvasquez commented on layouts/index.html:
```diff
@@ -494,29 +494,40 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
+                     </div>
+					 
+					<div class="col-12 col-md-8">
+					    <p>AlmaCare is an enterprise-grade AlmaLinux support service delivered by one of the founding sponsors of the AlmaLinux Project, TuxCare, a division of CloudLinux Inc.</p>
```
```suggestion
					    <p>Cybertrust is a provider of a comprehensive security certification services to various public and private entities globally, and offers comprehensive support for AlmaLinux users. Have also produced MiracleLinux for nearly 20 years.</p>
```

I think this is getting confused because we took TuxCare off the site, but adjusting for CyberTrust.

bennyvasquez commented:
@mattlasheboro as long as you're okay with my adjustments, you can commit them in and then we'll do one final check to make sure it looks good before merging!

mattlasheboro commented:
I believe everything is right now? I also changed the link for Cybertrust because it seemed incorrect, but if I made a mistake there let me know and I'll swap it back.

bennyvasquez commented:
Looks good to me. Thank you, @mattlasheboro! This is awesome.

mattlasheboro commented:
Great, thank you! I'm happy to help how I can.
